src/main/java/uk/gov/hmcts/reform/et/syaapi/service/CaseService.java

```diff
@@ -92,7 +92,7 @@ public CaseDetails createCase(String authorization, String caseType, String even
         return ccdApiClient.submitForCaseworker(
             authorization,
             s2sToken,
-            userDetails.getId(),
+            "123456",
```

Comment:
Can you change this to userDetails.getId() ?

Reply:
Fixed